klyshko-provisioner/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+FROM ghcr.io/carbynestack/openjdk:jre8-20210827
+
+ARG CLI_VERSION=0.2-SNAPSHOT-2336890983-14-a4260ab
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    gettext-base \
+    curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download Carbyne Stack CLI Jar at specified version
+RUN curl -o cs.jar -L https://github.com/carbynestack/cli/releases/download/$CLI_VERSION/cli-$CLI_VERSION-jar-with-dependencies.jar
```

Comment:
Alternative would be having Docker download the file for you, the calls to apt and to curl would not be needed:
```Dockerfile
ADD https://github.com/carbynestack/cli/releases/download/$CLI_VERSION/cli-$CLI_VERSION-jar-with-dependencies.jar ./cs.jar
```